helm-charts/hbase-chart/templates/meta/_zkscript.tpl

```diff
@@ -17,8 +17,26 @@ function shutdown() {
   echo "Stopping Zookeeper"
   $HBASE_HOME/bin/hbase-daemon.sh stop zookeeper
   if [ $exit_status != 0 ]; then
-	  echo "Leader stopped, sleeping for 60 seconds"
-	  sleep 60
+    IFS=$'\n' read -d '' -ra ZKs <<< $($HBASE_HOME/bin/hbase zkcli config 2> /dev/null  | grep server. | grep -o '=.*:' | cut -d : -f 1 | cut -c 2-)
+    function leaderElection() {
+       for zk in "${ZKs[@]}"; do
+           myhost=$(echo $zk 2181)
+           if [[ $zk != $(hostname -f) && $(echo "stat" | nc $myhost | grep "Mode: leader") ]]; then
```

Comment:
take this out to a variable `$(echo "stat" | nc $myhost | grep "Mode: leader")` so that it can be logged

Reply:
addressed